src/services/llm_service.py

```diff
@@ -89,7 +93,10 @@ def _load_prompt(self, prompt_path: str) -> str:
             raise
 
     def create_ai_chain(
-        self, prompt_path: str, additional_tools: Optional[List[BaseTool]] = None
+        self,
+        prompt_path: str,
+        additional_tools: Optional[List[BaseTool]] = None,
+        force_use_tool: str = "none",
```

Comment:
Why is this "none"?
We can check if the value is included or not.

"""
if force_use_tool:
"""

Also, I prefer something like "tool_to_use"

Reply:
Done